perf_tests/locust.conf

```diff
@@ -0,0 +1,32 @@
+# Location of the locustfile. This works as the entrypoint
+locustfile = random_index_and_tensor_search.py
+
+# Whether to run with or without the GUI
+headless = true
+
+# Host to load test
+host = http://localhost:8882
+
+# Total number of simulated users
+users = 5
```

Comment:
should this be set to the github workflow input?

Reply:
This locust file is just for specifying default values. The workflow input will override these.